src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -197,6 +245,11 @@ type FlagActionCounts {
   flagged for.
   """
   reasons: FlagReasonActionCounts!
+
+  """
+  flags is the actual Flags that were left by the Users or the system.
+  """
+  flags(first: Int = 10, after: Cursor): FlagsConnection!
```

Comment:
Wouldn't it makes better sense to put this directly on the `Comment`?

Reply:
We can't predict how many flags will be attached to the comment, so embedding them there would be unwise.